beetsplug/convert.py

```diff
@@ -298,21 +307,46 @@ def convert_item(self, dest_dir, keep_new, path_formats, fmt,
                                    util.displayable_path(original))
                     util.move(item.path, original)
 
+            linked = False
+
             if should_transcode(item, fmt):
                 try:
                     self.encode(command, original, converted, pretend)
                 except subprocess.CalledProcessError:
                     continue
             else:
-                if pretend:
-                    self._log.info(u'cp {0} {1}',
```

Comment:
If you keep the `if pretend:` here, you should be able to change the `cp {0} {1}` to `{2} {0} {1}` or `{0} {1} {2}` and pass the command name

Reply:
This is a very good idea.